src/main/java/edu/ucla/library/iiif/auth/delegate/hauth/HauthToken.java

```diff
@@ -12,10 +12,10 @@
     /**
      * An authorization header.
      * <p>
-     * N.B.: Cantaloupe's request context uses a map with lowercase keys.
+     * N.B.: The keys in Cantaloupe's request context are capitalized with HTTP/1.1, and lowercased with HTTP/2.
```

Comment:
Huh, interesting.

Reply:
I think it's unfortunate that they couldn't have just been made the same case, but I imagine it may be too late to make the HTTP/1.1 ones lowercase. Either way I do wish this was documented in the Cantaloupe API docs because I a few hours figuring this out.